Specifications/Language/3_Expressions/CopyAndUpdateExpressions.md

```diff
@@ -56,8 +52,16 @@ For example, if `arr` contains an array `[0,1,2,3]`, then
 
 Copy-and-update expressions allow efficient creation of new arrays based on existing ones. 
 The implementation for copy-and-update expressions avoids copying the entire array 
-but merely duplicates the necessary parts to achieve the desired behavior, and performs an in-place modification if possible. 
-Suitable ways to initialize an array via, e.g., an initialization function or similar means are provided by the standard libraries. Array initialization via a call to such a core function does not incur additional overhead due to immutability. 
+but merely duplicates the necessary parts to achieve the desired behavior, and performs an in-place modification if possible. Array initializations hence do not incur additional overhead due to immutability.
+
+The `Microsoft.Quantum.Arrays` namespace provides and arsenal of convenient tools for array creation and manipulation. 
+For instance, the function `ConstantArray` creates an array of the specified length and initializes each item to a given value. 
 
 
+Copy-and-update expressions are a convenient way to construct new arrays on the fly;
+the following expression, e.g., evaluates to an array with all items set to `PauliI`, except the item at index `i`, which is set to `PauliZ`:
+```qsharp
+ConstantArray(n, PauliI) w/ i <- PauliZ
```

Comment:
💕 This is easily one of my favorite uses of `w/`, I love seeing it highlighted here.